src/components/specific/models/models-manager/models-table/ModelsTable.vue

```diff
@@ -91,11 +122,23 @@ export default {
       { immediate: true }
     );
 
+    const background = computed(
+      () =>
+        `var(--color-silver-light) url("/static/modelsManager/menuAnimation/${
+          Object.values(MAIN_LANGUAGES).find(lang => lang === locale.value) ||
+          DEFAULT_LANGUAGE.EN
```

Comment:
You can use `$i18n.fallbackLocale` to get the default locale

Reply:
nice tip, thank you